test/simulator/timestamp/simulator_interface/simulator_interface.cpp

```diff
@@ -158,21 +104,82 @@ interface_rollback_transaction()
 }
 
 void
-interface_set_timestamp()
+interface_timestamp_transaction()
 {
 }
 
 void
-interface_timestamp_transaction()
+interface_session_query_timestamp()
 {
 }
 
 void
-interface_timestamp_transaction_uint()
+print_rules()
 {
 }
 
-void
-print_rules()
+int
+main(int argc, char *argv[])
 {
+    bool exit = false;
+
+    do {
+        std::cout << std::endl;
+
+        std::cout << "1: Session Management " << std::endl;
+        std::cout << "2: [Conn] set_timestamp()" << std::endl;
+        std::cout << "3: [Conn] query_timestamp()" << std::endl;
+        std::cout << "4: [Session] begin_transaction()" << std::endl;
+        std::cout << "5: [Session] commit_transaction()" << std::endl;
+        std::cout << "6: [Session] prepare_transaction()" << std::endl;
+        std::cout << "7: [Session] rollback_transaction()" << std::endl;
+        std::cout << "8: [Session] timestamp_transaction()" << std::endl;
+        std::cout << "9: [Session] query_timestamp()" << std::endl;
+        std::cout << "10: print rules for timestamps" << std::endl;
+        std::cout << "11: exit" << std::endl;
+
+        int choice = choose_num(1, 11, "timestamp_simulator >>");
```

Comment:
11 would become the size of the vector.

Reply:
Updated in [d3b07ca](https://github.com/wiredtiger/wiredtiger/pull/8417/commits/d3b07cad2974c45eb19a02631df03808cff9758d)